src/embed.js

```diff
@@ -118,11 +118,11 @@ function embed(el, spec, opt, callback) {
 
   var runtime = vega.parse(spec, opt.config); // may throw an Error if parsing fails
   try {
-    var view = new vega.View(runtime)
+    var view = new vega.View(runtime, opt.viewConfig)
```

Comment:
Hmm, why not just pass in `opt` and remove the calls to render below? 

Regardless, please add some documentation to the readme.